jboss-kie-workbench/added/launch/jboss-kie-workbench.sh

```diff
@@ -116,6 +116,11 @@ function configure_openshift_enhancement() {
     local kscPreferKieService=$(find_env "KIE_CONTROLLER_OPENSHIFT_PREFER_KIESERVER_SERVICE" "false")
     local kscTemplateCacheTTL=$(find_env "KIE_CONTROLLER_TEMPLATE_CACHE_TTL" "60000")
 
+    # Business Central is unified for RHDM and RHPAM; For rhpam-decisioncentral image needs to be set
+    # org.kie.workbench.profile to FORCE_PLANNER_AND_RULES
+    if [[ $JBOSS_PRODUCT =~ rhdm\-decisioncentral? ]]; then
+        JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.kie.workbench.profile=FORCE_PLANNER_AND_RULES"
```

Comment:
For RHPAM the org.kie.workbench.profile should be set to FORCE_FULL, to have same configuration as in on premise.